lib/muffin_man/fulfillment_outbound/v20200701.rb

```diff
@@ -0,0 +1,57 @@
+module MuffinMan
+  module FulfillmentOutbound
+    class V20200701 < SpApiClient
+
+      # To get outbound fulfillment preview
+      # @param [MuffinMan::RequestHelpers::OutboundFulFillment::FulfillmentPreviewRequest] fulfillment_preview_request in the form of object
+      def get_fulfillment_preview(fulfillment_preview_request)
+        @local_var_path = "/fba/outbound/2020-07-01/fulfillmentOrders/preview"
+        @request_body = fulfillment_preview_request.to_camelize
+        @request_type = "POST"
+        call_api
+      end
+
+      # To create outbound fulfillment order
+      # @param [MuffinMan::RequestHelpers::OutboundFulFillment::FulfillmentOrderRequest] fulfillment_order_request in the form of object
+      def create_fulfillment_orders(fulfillment_order_request)
```

Comment:
```suggestion
      def create_fulfillment_order(fulfillment_order_request)
```
The action name for this is [createFulfillmentOrder](https://developer-docs.amazon.com/sp-api/docs/fulfillment-outbound-api-v2020-07-01-reference#createfulfillmentorder), so we want to keep the naming consistent

Reply:
Added the changes